tests/unit/s2n_mem_usage_test.c

```diff
@@ -17,15 +17,15 @@
     /* FreeBSD requires POSIX compatibility off for its syscalls (enables __BSD_VISIBLE)
      * Without the below line, <sys/user.h> cannot be imported (it requires __BSD_VISIBLE) */
     #undef _POSIX_C_SOURCE
-    /* clang-format off */
+/* clang-format off */
```

Comment:
nit - why was this changed? This looks more strange now I think.

Reply:
I believe the original comments can't pass the clang-format test. I ran `clang-format -i ./tests/unit/s2n_mem_usage_test.c` and it changes the test to look like this way.